promql/engine.go

```diff
@@ -2784,6 +2784,7 @@ type groupedAggregation struct {
 	seen              bool // Was this output groups seen in the input at this timestamp.
 	hasFloat          bool // Has at least 1 float64 sample aggregated.
 	hasHistogram      bool // Has at least 1 histogram sample aggregated.
+	abandonHistogram  bool // If true, group has seen mixed exponential and custom buckets, or incompatible custom buckets.
```

Comment:
Maybe let's call it "incompatibleHistograms"? The current name sounds like we'll kick out the histogram, and then we'll still get a valid result or something.

Reply:
Changed in 82bb35f